lib/models/base.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+import (
+	"time"
+)
+
+type Base struct {
+	Id        int64 `gorm:"unique;primaryKey"`
```

Comment:
Gorm's convention using `ID` https://gorm.io/docs/models.html#gorm-Model

Feel free to check with the FK also 🖖
https://gorm.io/docs/belongs_to.html